0.4.1 fixes (#44)
ocwg/spec: spec/v0.4.1-draft/spec.md

Conceptually, a node is a rectangle (bounding box) on the canvas, often displaying some content (resource).
A _Node_ is an `object` with the following properties:

| Property       | JSON Type | OCIF Type                         | Required     | Contents                            | Default     |
|----------------|-----------|-----------------------------------|--------------|-------------------------------------|-------------|
| `id`           | `string`  | [ID](#id)                         | **required** | A unique identifier for the node.   | n/a         |
| `position`     | `array`   | number[]                          | recommended  | Coordinate as (x,y) or (x,y,z).     | [0,0]       |
| `size`         | `array`   | number[]                          | recommended  | The size of the node per dimension. | `[100,100]` |
| `resource`     | `string`  | [ID](#id)                         | optional     | The resource to display             |             |

> | `resource-fit` | `string`  | enum, see below                   | optional     | Fitting resource in item            | `contain`   |

Reviewer: ```suggestion
| `resourceFit` | `string`  | enum, see below                   | optional     | Fitting resource in item            | `contain`   |
```

  - Resource can be empty, in which case a node is acting as a transform for other nodes.
  - z-ordering: The resource is to be rendered behind the node. For example, if the node has a rectangular border (or [oval](#oval)), that border would be rendered in front of the resource.

> - **resource-fit**: Given a node with dimensions 100 (height) x 200 (width) and a bitmap image (e.g., a .png) with a size of 1000 x 1000.

Reviewer: ```suggestion
- **resourceFit**: Given a node with dimensions 100 (height) x 200 (width) and a bitmap image (e.g., a .png) with a size of 1000 x 1000.
```

  How should this image be displayed? We re-use some options from CSS ([object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) property):

  - `none`: All pixels are displayed in the available space unscaled. The example would be cropped down to the 100 x 200 area top-left. No auto-centering.

>   - `keep-width`: Scaled by keeping the aspect ratio, so that the image width matches the item width. This results in the image being displayed at a scale of `0.5`, so that it is 200 px wide and 200 px heigh. The top half of the image is visible.
>   - `keep-height`: Scaled by keeping the aspect ratio, so that the image height matches the item height. This results in the image being displayed at a scale of `0.1`, so that it is 100 px high and 100 px wide. The image is now fully visible, but there are boxes of empty space left and right of the image.

Reviewer: ```suggestion
  - `keepWidth`: Scaled by keeping the aspect ratio, so that the image width matches the item width. This results in the image being displayed at a scale of `0.5`, so that it is 200 px wide and 200 px heigh. The top half of the image is visible.
  - `keepHeight`: Scaled by keeping the aspect ratio, so that the image height matches the item height. This results in the image being displayed at a scale of `0.1`, so that it is 100 px high and 100 px wide. The image is now fully visible, but there are boxes of empty space left and right of the image.
```

Reviewer 2: keepHeight -> containY
keepWidth -> containX
